index.js

```diff
@@ -59,6 +61,12 @@ function bootApp() {
     app.set("trust proxy", 1)
   }
 
+  app.use(ipfilter(IP_BLACKLIST.split(','), {
+    allowedHeaders: ['x-forwarded-for'],
+    log: false,
+    mode: 'deny'
+  }))
```

Comment:
Not sure if you think we’ll always have something in `IP_BLACKLIST` but otherwise it might be nice to not add this middleware at all in the case it’s blank?